pykickstart/commands/timezone.py

```diff
@@ -141,11 +175,8 @@ def __init__(self, *args, **kwargs):
         F18_Timezone.__init__(self, *args, **kwargs)
         self.ntpservers = kwargs.get("ntpservers", list())
 
-    def _getParser(self):
-        op = FC6_Timezone._getParser(self)
-        op.add_argument("--nontp", action="store_true", default=False)
-        op.add_argument("--ntpservers", type=commaSplit)
-        return op
+#todo: I've removed _getParser() but this method also looks like above
+# something to do with lists and sets
```

Comment:
This is fine - I don't see any reason why there's the duplicate _getParser method.  The distinct parse methods need to stay, though.  We made a change where you can have the same NTP server listed multiple times for some reason, whereas other releases do not allow for that.